languages/java8/template/src/main/java/com/algorithmia/algorithm/Algorithm.java

```diff
@@ -0,0 +1,54 @@
+package com.algorithmia.algorithm;
+
+import com.algorithmia.development.*;
+import com.algorithmia.*;
+
+class Algorithm {
+    public String apply(String input) throws RuntimeException {
+        return "hello ".concat(input);
+    }
+
+    public static void main(String[] args) {
+        Algorithm defs = new Algorithm();
+        Handler algo = new Handler<>(defs.getClass(), defs::apply);
+        algo.serve();
+    }
+}
+
+
+/**
+ * This class below describes a more advanced template.
```

Comment:
I feel like this comment is a little long. Can you just always have an empty load step explaining what to use it for

Reply:
I like that idea honestly, are you comfortable with that style @kennydaniel?